packages/extension/src/newtab/ShortcutLinks.tsx

```diff
@@ -315,73 +247,30 @@ export default function ShortcutLinks({
     });
   };
 
-  const ShortcutV1 = () => {
-    return (
-      <div
-        className={classNames(
-          'hidden tablet:flex',
-          shouldUseListFeedLayout ? 'mx-6 mb-3 mt-1' : '-mt-2 mb-5',
-        )}
-      >
-        {shortcutLinks?.length ? (
-          <>
-            {shortcutLinks.map((url) => (
-              <ShortcutV1Item key={url} url={url} onLinkClick={onLinkClick} />
-            ))}
-            <Button
-              variant={ButtonVariant.Tertiary}
-              size={ButtonSize.Small}
-              icon={<MenuIcon className="rotate-90" secondary />}
-              onClick={onMenuClick}
-              className="mt-2"
-            />
-          </>
-        ) : (
-          <>
-            <ShortCutV1Placeholder initialItem onClick={onOptionsOpen} />
-            {Array.from({ length: 5 }).map((_, index) => (
-              /* eslint-disable-next-line react/no-array-index-key */
-              <ShortCutV1Placeholder key={index} onClick={onOptionsOpen} />
-            ))}
-            <Button
-              variant={ButtonVariant.Tertiary}
-              onClick={onV1Hide}
-              size={ButtonSize.Small}
-              icon={<ClearIcon secondary />}
-              className="mt-2"
-            />
-          </>
-        )}
-      </div>
-    );
-  };
-
   return (
     <>
       {isShortcutsV1 ? (
-        <ShortcutV1 />
+        <ShortcutLinksV1
```

Comment:
Maybe we should create a separate file for each version, just to make it easier to maintain and review.

Reply:
Yeah think it might be time.
During experiment we normally don't but seeing it's a stale experiment (will live for a while) we probably should abstract into two files.

Reply:
Hi @sshanzel , thanks! Totally agree with you, was very hard time to understand how this component works.
I'm making a new directory with all component-related files and split v1 and control